src/Google/Ads/GoogleAds/Util/FieldMasks.php

```diff
@@ -247,4 +254,14 @@ private static function isWrapperType(Descriptor $descriptor)
     {
         return in_array($descriptor->getClass(), self::$WRAPPER_TYPES);
     }
+
+    // TODO: We can remove this function when it's supported in google/gax-php.
```

Comment:
Any gax-php issue # you point to? Could you create a new issue in the project so we can keep track of this?

Reply:
From what you pointed out, let's discuss offline to find the conclusion before getting back to this again.